pkgs/applications/audio/melmatcheq.lv2/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, xorg, xorgproto, cairo, lv2, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "MelMatchEQ.lv2";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "brummer10";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1s805jgb9msxfq9047s7pxrngizb00w8sm4z94iii80ba65rd20x";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    xorg.libX11 xorgproto cairo lv2
+  ];
+
+  installFlags = [ "INSTALL_DIR=$(out)/lib/lv2" ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/brummer10/MelMatchEQ.lv2";
+    description = "a profiling EQ using a 26 step Mel Frequency Band";
+    maintainers = [ maintainers.magnetophon ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ magnetophon ];
```

Reply:
Thanks for the review.
May I ask why you propose this change?
I can see the point of ``with`` as part of a [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) strategy, but I don't see the point in this case, as there is only one maintainer.

Reply:
If there are two in the future you don't need to change it.